app/src/components/comment/Comment.css

```diff
@@ -0,0 +1,12 @@
+.Comment {
+    background: var(--color-black);
+    margin-top: 2px;
+    padding: 6px 32px;
+}
+
+.Comment__header {
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+    justify-content: space-between;
+}
```

Comment:
I think you might need to update your linter or .vscode or tslint or whatever to insert the linebreak at the end of file

Reply:
Yes :100: the husky pre-commit hook hasn't been working, thanks :pray: